### ISSUE TITLE
SQL export not working

### ISSUE THREAD

**Issue by Davincible** [sosedoff/pgweb#566]
I have pgweb running in Kubernetes locally with a port forward allowing me to access it.

When I try "Export to SQL" on any random table, it opens up a new tab `about:blank`. All the other export options do work fine, and open up a popup allowing me to select the download destination. 

Since the other functions work fine, I would expect the SQL export to also work. 

I'm not sure if this bug is specific to my set up, or pgweb in general

**Comment by sosedoff** [comment]
Im suspecting your container does not have `psql` installed, is that correct? Pgweb uses psql under the hood to generate sql dumps.

**Comment by Davincible** [comment]
I'm using the `sosedoff/pgweb` image. After shelling in it seems to be installed

**Comment by sosedoff** [comment]
I think you can start the container with `--debug` option, it'll display an error. I would like to see your logs when you run export anyway, hard to know what went wrong without it.

**Comment by Davincible** [comment]
Changed my deployment to
```yaml
          image: sosedoff/pgweb
          command: 
            - /usr/bin/pgweb
          args:
            - --bind=0.0.0.0
            - --listen=8081
            - --debug

```

Not so useful logs

```
pgweb-5d5fdf8876-rs5gc pgweb 2022/06/22 12:24:46 Request params: <nil> map[_session_id:[cf0797b0-1774-c78c-bbab-fa6cd1ce342d] table:["public"."Proxies"]]
pgweb-5d5fdf8876-rs5gc pgweb [GIN] 2022/06/22 - 12:24:46 | 400 |   15.402985ms |       127.0.0.1 | GET      "/api/export?table=%22public%22.%22Proxies%22&_session_id=cf0797b0-1774-c78c-bbab-fa6cd1ce342d"
```

**Comment by Davincible** [comment]
Ah, after manually curl'ing the export command I get the following:
```json
{
   "error":"error: exit status 1. output: pg_dump: error: server version: 14.3 (Ubuntu 14.3-1.pgdg22.04+1); pg_dump version: 13.6\npg_dump: error: aborting because of server version mismatch\n",
   "status":400
}
```

**Comment by Davincible** [comment]
After running the exact `Dockerfile` locally it does install `pg_dump` v14, I think at the time you build the container it installed the older version. Rebuilding the `sosedoff/pgweb` container should fix this I think

**Comment by Davincible** [comment]
@sosedoff could you please rebuild and push the docker container so it updates the pg_dump version?

**Comment by sosedoff** [comment]
While i can rebuild the image, i think we should also pin the postgres client libraries to a specific version, like 14.

**Comment by Davincible** [comment]
You could mention them in the image tags (e.g. like Timescaldb mentions both the timescale version and pg version in their [tags](https://hub.docker.com/r/timescale/timescaledb/tags)). Since they're all backward compatible I think someone rarely would want to  use an older version specifically

**Comment by sosedoff** [comment]
I would like to avoid having this kind of "matrix" of builds if possible, however you can open up a new issue on that topic, i can work something out for new releases.

**Comment by Davincible** [comment]
Fair enough. Perhaps you could release an `0.11.12` image just to upgrade Postgres

**Comment by sosedoff** [comment]
Good call - i just implemented automation for docker images , so 0.11.12 could be a good candidate to test that out.
